packages/design-system-tailwind-preset/CHANGELOG.md

```diff
@@ -7,4 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/metamask-design-system/
+## [0.1.0]
+
+### Added
+
+- Initial release.
```

Comment:
Intial release of the tailwind preset that we can use in the portfolio to easily align with future updates of the design tokens